src/protocol/mod.rs

```diff
@@ -715,20 +712,22 @@ mod tests {
         ]);
         let limit = WebSocketConfig { max_message_size: Some(10), ..WebSocketConfig::default() };
         let mut socket = WebSocket::from_raw_socket(WriteMoc(incoming), Role::Client, Some(limit));
-        assert_eq!(
-            socket.read_message().unwrap_err().to_string(),
-            "Space limit exceeded: Message too big: 7 + 6 > 10"
-        );
+
+        match socket.read_message() {
+            Err(Error::Capacity(CapacityError::MessageTooLong { size: 13, max_size: 10 })) => {}
+            _ => panic!(),
+        }
```

Comment:
Oh, I forgot to mention it in my previous review - there were in total 3 instances where matches!() should be used - this one plus one below.

Reply:
Ah, my apologies, all fixed now.